src/App.test.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 
+import { useDispatch, useSelector } from 'react-redux';
+
 import { render } from '@testing-library/react';
 
 import App from './App';
 
-test('App', () => {
-  const { getByText } = render((
-    <App />
-  ));
+jest.mock('react-redux');
+
+describe('App에서', () => {
+  const dispatch = jest.fn();
+
+  useDispatch.mockImplementation(() => (dispatch));
+  useSelector.mockImplementation((selector) => selector({
+    taskTitle: '',
+    tasks: [
+      { id: 100, title: '밥 먹기' },
+      { id: 101, title: '누워있기' },
+    ],
+  }));
 
-  expect(getByText(/추가/)).not.toBeNull();
+  it('어플리케이션 제목(할 일)를 보여준다.', () => {
+    const { getByText } = render((
+      <App />
+    ));
```

Comment:
```suggestion
    const { getByText } = render(<App />);
```

간단한 건 한줄로 작성해줘도 괜찮은 것 같은데 어떠신가요?

줄바꿈을 하실 때도 자기만의 기준이 있으면 좋아요.

저 같은 경우 `props` 가 2개이상 넘어가면 줄바꿈을 해주는 편이에요.

`props` 가 각 한 줄로 보이는게 가독성에 좋다고 생각하기 떄문이에요. 개인적은 견해는 다를 수 있지만 나름 자기만의 논리가 있으면 좋을 것 같아요!

Reply:
넵넵! 수정했습니다!👍